### PR TITLE
David/863 build networks

### LINKED ISSUE #863: Refactor build process to include networks dir and config toml
UI Version: 0.X.X

Description:

### PR DESCRIPTION
**Pull request by NodeGuy**
Closes #863

Description:

Network configuration files are now taken from the SDK.

### CODE REVIEW

**Comment by NodeGuy** [comment]
Our unit tests fail because they're expecting the network configuration to have the files `config.toml` and `basecoindversion.txt` but the network configuration from the SDK has only `genesis.json`.

What should we do?

**Comment by faboweb** [comment]
Note: Please pull people in, as I tend to skip WIP PRs.

**Comment by faboweb** [comment]
`basecoindversion.txt` just holds the version of the SDK. This can be generated or gathered differently.
The missing config.toml is bad.

**Comment by NodeGuy** [comment]
@mappum How should we handle this?

**Comment by mappum** [comment]
I would assume we could just build the `basecoindversion.txt` for each network based on the current version of the SDK for simplicity.

As for the `config.toml`, the important part Voyager gets from that is the peer addresses, and I'm not sure where to get those if they aren't in the SDK.

**Comment by NodeGuy** [comment]
> I would assume we could just build the basecoindversion.txt for each network based on the current version of the SDK for simplicity.

OK

> As for the config.toml, the important part Voyager gets from that is the peer addresses, and I'm not sure where to get those if they aren't in the SDK.

Got it.  I'll automate the manual process describe at https://github.com/cosmos/cosmos-sdk/tree/master/cmd/gaia/testnets.

**Comment by faboweb** [comment]
The main thread tests read in some configs. In this PR all networks where removed. I added a mock configuration for the test.
And it mocks "fs-extra" and in one file "fs" was used. We probably should just mock "fs" as well.

**Comment by NodeGuy** [comment]
Thank you.

File: CHANGELOG.md [hunk 1]
```diff
@@ -83,7 +84,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * E2E test maybe fix @okwme
 * Login in to restricted page bug @okwme
 * Send with an empty wallet bug @okwme
-* Readme formatting @okwme
+  <<<<<<< HEAD
```
Comment (faboweb) [inline CHANGELOG.md]: Merge conflict?

Comment (NodeGuy) [reply CHANGELOG.md]: Oh dear, thank you.

**Pull request merged**